event-api/src/entity/CapteurArchiveEntity.php

```diff
@@ -0,0 +1,166 @@
+<?php
+// src/Entity/CapteurArchiveEntity.php
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="CapteurArchive")
+ */
+#[ORM\Entity]
+#[ORM\Table(name: "Capteur_Archive")]
+class Capteur_Archive
+{
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="integer")
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    #[ORM\Id]
+    #[ORM\Column(type: "integer")]
+    #[ORM\GeneratedValue(strategy: "AUTO")]
+    private $id;
+
+    /**
+     * @ORM\Column(type="integer")
+     */
+    #[ORM\Column(type: "integer", name: "reservation_id")]
```

Comment:
Les clés étrangères doivent être déclaré en many-to-one ou autre:

https://www.doctrine-project.org/projects/doctrine-orm/en/3.1/reference/association-mapping.html

Doctrine ne pourra pas faire le lien autrement.

Reply:
![image](https://github.com/MathysHaubert/Event-it-API/assets/143439294/bc0065f2-47c6-4cc4-a6dc-a8cf3bc143b0)
Et c'est quoi ça ?